rosdep/base.yaml

```diff
@@ -2806,6 +2806,14 @@ liblapack-dev:
 liblapacke-dev:
   debian: [liblapacke-dev]
   ubuntu: [liblapacke-dev]
+liblcm-dev:
+  debian: [liblcm-dev]
+  fedora: [liblcm-devel]
+  ubuntu: [liblcm-dev]
+liblcm1:
```

Comment:
I'd be more inclined to make this key: `liblcm`.  If it ever gets a newer version in the future, it will be more forwards-compatible that way.

Reply:
I think it's somewhat dependent on whether the expectation is that future releases contain liblcm2 and not liblcm1. Since the Fedora package names don't include the version I think this is fine as is although my preference would be to follow the Debian package naems for consistency with other rosdep keys.

Reply:
I'm fine with it either way